src/dos/program_mousectl.cpp

```diff
@@ -541,24 +541,25 @@ void MOUSECTL::AddMessages()
 	        "Examples:\n"
 	        "  [color=green]mousectl[reset] [color=white]DOS[reset] [color=white]COM1[reset] -map    ; asks user to select mice for a two player game");
 
-	MSG_Add("PROGRAM_MOUSECTL_SYNTAX", "Wrong command syntax.");
 	MSG_Add("PROGRAM_MOUSECTL_SYNTAX_PATTERN",
-	        "Wrong syntax, only ASCII characters allowed in pattern.");
+	        "Incorrect syntax, only ASCII characters allowed in pattern.\n");
 	MSG_Add("PROGRAM_MOUSECTL_SYNTAX_SENSITIVITY",
-	        "Wrong syntax, sensitivity needs to be in -999 to +999 range.");
+	        "Incorrect syntax, sensitivity needs to be in -999 to +999 range.\n");
 	MSG_Add("PROGRAM_MOUSECTL_SYNTAX_DUPLICATED",
-	        "Wrong syntax, duplicated mouse interfaces.");
+	        "Incorrect syntax, duplicated mouse interfaces.\n");
 	MSG_Add("PROGRAM_MOUSECTL_SYNTAX_MIN_RATE",
-	        "Wrong syntax, sampling rate has to be one of:\n%s");
+	        "Incorrect syntax, sampling rate has to be one of:\n%s\n");
 
 	MSG_Add("PROGRAM_MOUSECTL_MAPPING_NO_MOUSE",
-	        "Mapping not available in no-mouse mode.");
-	MSG_Add("PROGRAM_MOUSECTL_NO_INTERFACES", "No mouse interfaces available.");
+	        "Mapping not available in no-mouse mode.\n");
+	MSG_Add("PROGRAM_MOUSECTL_NO_INTERFACES",
+	        "No mouse interfaces available.\n");
 	MSG_Add("PROGRAM_MOUSECTL_MISSING_INTERFACES",
-	        "Mouse interface not available.");
-	MSG_Add("PROGRAM_MOUSECTL_NO_PHYSICAL_MICE", "No physical mice detected.");
+	        "Mouse interface not available.\n");
+	MSG_Add("PROGRAM_MOUSECTL_NO_PHYSICAL_MICE",
+	        "No physical mice detected.\n");
 	MSG_Add("PROGRAM_MOUSECTL_NO_MATCH",
-	        "No available mouse matching the pattern found.");
+	        "No available mouse matching the pattern found.\n");
```

Comment:
No available mouse found matching the pattern.

Reply:
OK, changed.